### DISCUSSION TITLE
arabic text

### DISCUSSION THREAD

**Discussion by lazezo2** [pndurette/gTTS#397] [Q&A] [answered]
Arabic text give zero duration mp3 file

```
from gtts import gTTS
tts = gTTS('اهلا')
tts.save('hello.mp3')
```

any help?

**Comment by pndurette** [comment, marked as answer]
By default it's trying to read that in English—which it can't—so you need to specify the language to use, in your case:

```diff
from gtts import gTTS
-tts = gTTS('اهلا')
+tts = gTTS('اهلا', lang='ar')
tts.save('hello.mp3')
```

Using `ar` for Arabic should do the trick!

**Reply by lazezo2** [reply]
thanks, that's working

**Comment by AMINEHAIRICHE** [comment]
اهلا كيف الحال
